SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -260,6 +260,9 @@ int64 USpatialActorChannel::ReplicateActor()
 		Bunch.bReliable = true; // Net temporary sends need to be reliable as well to force them to retry
 	}
 
+	// Use spatial's bCreatingNewEntity to inform the replication flags.
```

Comment:
Slight driveby, but could we remove the `OpenPacketId` stuff above?

Reply:
So a slight question with this would be that there is some code within that logic regarding a FOutBunch. Since the OpenPacketId case is always reached, it would be effectively the same to take the code out of the if statement. It may be the case that we don't need the Bunch at all however, but it is passed to a virtual OnSerializeNewActor method, and I'm currently unclear on how this would be intended to be used, and whether removing this entirely would lead to unexpected consequences down the line

Reply:
All bunch usage only remains to satisfy existing interfaces. And we're now overriding the RepFlags.bNetInitial, so removing the OpenPacketId clause is safe to remove.